docs/NSR.md

```diff
@@ -223,7 +221,7 @@ CA Infrastructure MUST be in a Physically Secure Environment.
 CA Infrastructure and Network Equipment MUST be implemented and configured to authenticate and encrypt connections:
```

Comment:
Connections to- and within the CA infrastructure MUST be authenticated and encrypted.